src/controllers/pods.ts

```diff
@@ -49,12 +50,13 @@ export default function (server: Hapi.Server, deps: Injector) {
   async function chargeForDuration (request: any): Promise<string> {
     const duration = request.query['duration'] || '3600'
 
-    const currencyPerSecond = getCurrencyPerSecond(deps)
+    console.log(typeof config)
+    const currencyPerSecond = getCurrencyPerSecond(config)
     log.debug('got post pod request. duration=' + duration)
+    const price = currencyPerSecond.times(new BigNumber(duration)).integerValue(BigNumber.ROUND_CEIL)
+    log.debug('price: ' + price.toString())
```

Comment:
might make sense to combine this with the log statement above so we have less cluttered logs.